ddl/index.go

```diff
@@ -524,6 +529,10 @@ func (w *addIndexWorker) fetchRowColVals(txn kv.Transaction, taskRange reorgInde
 			}
 
 			w.idxRecords = append(w.idxRecords, idxRecord)
+			if handle == taskRange.endHandle {
```

Comment:
Should we consider `taskRange.endIncluded` here?

Reply:
If !taskRange.endIncluded, we will not reach here when handle == taskRange.endHandle , we return in 525

Reply:
Could comment for this?

Reply:
Done.